### PR TITLE
full dev to release merge

### PR DESCRIPTION
**Pull request by sdolenc**
full dev to release merge
- we need the npm compileall fix in master for today's "phase2" deployment
- the other changes are benign, validated, and ready to move up. i don't think we should hold them back any longer (especially considering that many are improve our testing or finish bringing us up to parity with ficus.1)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ branches:
   - oxa/master.fic
   - oxa/dev.fic
   - oxa/devfic
+  - oxa/release.fic
+  - firstHalf4
```
Comment (btelnes) [inline .travis.yml]: Remove firstHalf4

Comment (sdolenc) [reply .travis.yml]: good point. fixed with most recent commit. please approve at your leisure

**Comment by sdolenc** [comment]
@btelnes , i made your requested change. please re-review when you get the chance

**Comment by sdolenc** [comment]
closing this PR in favor of isolated merges #62  and #63

**Pull request closed without merging**